scripts/us_bjs/nps/nps_statvars.mcf

```diff
@@ -0,0 +1,979 @@
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_LocallyOperatedLocalFacility_Male
+correctionalFacilityOperator: dcs:LocallyOperated
+correctionalFacilityLocation: dcs:Local
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_FederallyOperatedFacility_Male
+correctionalFacilityOperator: dcs:FederallyOperated
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_StateOperatedOutOfStateFacility_Male
+correctionalFacilityOperator: dcs:StateOperated
+correctionalFacilityLocation: dcs:OutOfState
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Custody_NotAUSCitizen_Male
+citizenship: dcs:NotAUSCitizen
+correctionalFacilityOperator: dcs:StateOperated&FederallyOperated&PrivatelyOperated
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnCustody
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Custody_NotAUSCitizen_MaxSentenceGreaterThan1Year_Male
+prisonSentenceStatus: dcs:Sentenced
+maxPrisonSentence: [2 - Years]
+citizenship: dcs:NotAUSCitizen
+correctionalFacilityOperator: dcs:StateOperated&FederallyOperated&PrivatelyOperated
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnCustody
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Custody_NotAUSCitizen_MaxSentence1YearOrLess_Male
+prisonSentenceStatus: dcs:Sentenced
+maxPrisonSentence: [- 1 Years]
+citizenship: dcs:NotAUSCitizen
+correctionalFacilityOperator: dcs:StateOperated&FederallyOperated&PrivatelyOperated
+gender: schema:Male
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnCustody
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Custody_NotAUSCitizen_Unsentenced_Male
+prisonSentenceStatus: dcs:Unsentenced
+citizenship: dcs:NotAUSCitizen
+correctionalFacilityOperator: dcs:StateOperated&FederallyOperated&PrivatelyOperated
```

Comment:
Every PV should be encoded in the StatVar, otherwise it can lead to collisions going forward, say if someone had the same Node except without `correctionalFacilityOperator` and `institutionalization`